scriptmodules/system.sh

```diff
@@ -441,5 +441,5 @@ function platform_vero4k() {
     __default_cflags="-I/opt/vero3/include -L/opt/vero3/lib -O2 -mcpu=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations"
     __default_asflags=""
     __default_makeflags="-j4"
-    __platform_flags="arm armv7 neon vero4k gles"
+    __platform_flags="arm armv7 neon mali vero4k gles"
```

Comment:
You can remove vero4k from platform flags btw as it's automatically included.

Reply:
Good point.